examples/burial-form/MilitaryServiceHistory.jsx

```diff
@@ -1,32 +1,21 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import {
   DateField,
   Page,
-  TextField,
-  DebuggerView
+  TextField
 } from '@department-of-veterans-affairs/va-forms-system-core';
 import { useFormikContext } from 'formik';
+import { isBeforeDate } from './utils';
 
 export default function MilitaryServiceHistory(props) {
-  useEffect(() => {
-    window.scrollTo({
-      top: 0,
-      left: 0,
-      behavior: "smooth"
-    });
-  });
 
   const state = useFormikContext();
 
-  const isBeforeStartDate = (value) => {
```

Comment:
refactored isBeforeStartDate into isBeforeDate since it's used in two separate places.